task/mgmt-gateway/src/main.rs

```diff
@@ -121,14 +119,21 @@ fn main() {
 
 struct NetHandler {
     net: Net,
-    tx_buf: &'static mut SerializedMessageBuf,
-    rx_buf: &'static mut SerializedMessageBuf,
+    tx_buf: &'static mut [u8; gateway_messages::MAX_SERIALIZED_SIZE],
+    rx_buf: &'static mut [u8; gateway_messages::MAX_SERIALIZED_SIZE],
     packet_to_send: Option<UdpMetadata>,
 }
 
 impl NetHandler {
-    fn new(buffers: &'static mut [SerializedMessageBuf; 2]) -> Self {
-        let [tx_buf, rx_buf] = buffers;
+    fn claim_static_resources() -> Self {
+        let tx_buf = mutable_statics! {
+            static mut NET_TX_BUF: [u8; gateway_messages::MAX_SERIALIZED_SIZE] =
```

Comment:
Nit: I think you can put both arrays into the same `mutable_statics!` call, saving an atomic.

Reply:
Ooh, thanks. I was doing this as `[[u8; N]; 2]` before; didn't realize the macro supported multiple statics.